docker/android/Makefile

```diff
@@ -17,11 +17,14 @@ SDK_PLATFORM_VERSION=$(call  __toolversion, android-sdk-platform)
 SDK_BUILD_TOOLS_VERSION=$(call  __toolversion, android-sdk-build-tools)
 
 # WARNING: Remember to change the tag when updating the image
-IMAGE_TAG = 1.1.0
+BASE_IMAGE_TAG = $(shell cd ../base && make get-image-tag)
```

Comment:
Oh, I see. So the hash depends on the specific versions we get for that particular image. Interesting solution.